### DISCUSSION TITLE
Can you guys pleasse revert 1.0.0 and move it to 2.0.0?

### DISCUSSION THREAD

**Discussion by Alevale** [cheeriojs/cheerio#4030] [General]
As it has been gathered in the last week, it seems like the release of the "stable" 1.0.0 version has broken a lot of dependencies that were relying on both, parts of the library to be exposed in a certain way, and the node version being older than 18.

Since that is not the case, could we please revert the changes and put them on a newer version (2.0.0 for example)

Since the package wasn't really under heavy development, although we were on a RC version of the 1.0.0 release many packages on the community asssumed that after 7 years the RC was just the stable version. That didn't seem to be the case, and with the new release things have broken.

This break, is forcing the community to have to write overrides in their packages (you would be forcing A LOT of people to have to hardcode things to a specific version)  `"overrides": { "cheerio": "1.0.0-rc.12" }`

Let's be practical about this and just make things easier for everyone. Although SEMVER does say that `RC` branches are unstable and they could have functionality dropped from them at any point. I would argue that something that has been released for 2 years (the latest `RC-12` hasn't changed in that time) isn't the pure definition of "under development".

@matthewmueller @fb55 @jugglinmike please, read the issues there's in this package, and release a new version that makes things more stable for everyone

- https://github.com/cheeriojs/cheerio/issues/3993
- https://github.com/cheeriojs/cheerio/issues/4014
- https://github.com/cheeriojs/cheerio/issues/4011
- https://github.com/Automattic/juice/pull/494
- https://github.com/neocotic/convert-svg/issues/109
- https://github.com/cheeriojs/cheerio/issues/4024

**Comment by Romej** [comment]
It took a while to figure out why things were breaking on our end, I don't understand why introduce such a breaking change between `rc.12` and `1.0.0`
